docs/install-include.rst

```diff
@@ -58,16 +58,17 @@ Select your preferences below and run the corresponding install commands:
 
               .. code-block:: bash
 
-                 pip install --upgrade mxnet
+                 pip install -U --pre "mxnet>=1.7.0b20200713, <2.0.0" -f https://sxjscience.github.io/KDD2020/
```

Comment:
Is there a more official way to install this than using https://sxjscience.github.io/KDD2020/?

Reply:
Currently the 1.7.0 vote hasn't been passed yet so we have to use this form temporarily.